example-get-started-cv/code/params.yaml

```diff
@@ -0,0 +1,17 @@
+base:
+  random_seed: 42
+
+data_split:
+  test_pct: 0.15
+
+train:
+  valid_pct: 0.1
+  arch: resnet18
+  img_size: 256
```

Comment:
@alex000kim I decreased this, so we can manually create a P.R. increasing the image size via Studio, WDYT?

Reply:
yup, looks good